client/src/helpers/APICalls/upload.ts

```diff
@@ -0,0 +1,24 @@
+import { AuthApiData } from '../../interface/AuthApiData';
+
+interface FetchOptions {
+  method: string;
+  body: FormData;
+  credentials: RequestCredentials;
+}
+
+const upload = async (): Promise<AuthApiData> => {
+  const formData = new FormData();
+  const fetchOptions: FetchOptions = {
+    method: 'POST',
+    body: formData,
+    credentials: 'include',
+  };
+
+  return await fetch('/upload-images', fetchOptions)
+    .then((res) => res.json())
+    .catch(() => ({
+      error: { message: 'Unable to connect to server. Please try again' },
+    }));
+};
```

Comment:
We're submitting an empty formData here. We should receive the formData from the UI component as a parameter to this function so we can send it to the backend 👍🏼

Reply:
@ugur-akin Kindly explain it more please, because when I passed formData as an argument in this function, it throws error